dora/core/client/fs/src/main/java/alluxio/client/file/ufs/UfsBaseFileSystem.java

```diff
@@ -241,6 +243,18 @@ public URIStatus getStatus(AlluxioURI path, final GetStatusPOptions options) {
     });
   }
 
+  @Override
+  public URIStatus getStatus(UfsUrl ufsPath, final GetStatusPOptions options) {
+    return callWithReturn(() -> {
+      // TODO(jiacheng): can this path contain scheme and authority?
+      String pathStr = ufsPath.asString();
+      UfsStatus ufsStatus = mUfs.get().getStatus(pathStr, GetStatusOptions.defaults()
+              .setIncludeRealContentHash(options.getIncludeRealContentHash()));
+      // TODO: avoid using AlluxioURI in this method and avoid string copies as much as possible
```

Comment:
i think you are not going to resolve this TODO in this PR. In that case, add your name to the TODO like `TODO(tony)`